providers/src/airflow/providers/ydb/CHANGELOG.rst

```diff
@@ -27,6 +27,10 @@
 Changelog
 ---------
 
+.. note::
+  This release removes YDBScanQueryOperator from this provider package.
+  At this point, YDBExecuteQueryOperator could load unlimited amount of rows, so no speific operator is needed.
```

Comment:
Typo in speific

Reply:
Our spellchecker is much more picky than maintainers :).